account_credit_control/models/credit_control_policy.py

```diff
@@ -90,16 +90,18 @@ def _move_lines_subset(self, controlling_date, model, move_relation_field):
         # The lines which are linked to this policy have to be included in the
         # run for this policy.
         # If another object override the credit_policy_id (ie. invoice after
-        add_objs = my_obj.search([('credit_policy_id', '=', self.id)])
+        add_objs = my_obj.with_context(active_test=False).search([
```

Comment:
Instead of this, on the first line do:
```python
my_obj = self.env[model].with_context(active_test=False)
```
only once

Reply:
added